qcore/gen_coords.py

```diff
@@ -5,9 +5,9 @@
 from subprocess import check_call
 import sys
 
-from qcore.config import qconfig
+from qcore.binary_version import get_unversioned_bin
 
-GEN_COORD_BIN = os.path.join(qconfig['tools_dir'], 'gen_model_cords')
+GEN_COORD_BIN = get_unversioned_bin('gen_model_cords')
```

Comment:
Not sure if this should be a constant? From what I can see it is only used in one place (unless it used in other scripts), so might as well just change that to the function call?